packages/synapse-react-client/src/components/TimelinePlot/TimelinePhase.tsx

```diff
@@ -49,6 +52,23 @@ export const getMaxDate = (timepoints: dayjs.Dayjs[]) => {
 }
 
 const getTimelineData = (timepointData: TimepointData, rowData: Row[]) => {
+  // If this phase has no data, return an empty data line
+  if (timepointData.timepoints.length == 0) {
+    return [
+      {
+        x: [dayjs().format()],
+        y: [0.5],
+        mode: 'lines',
+        line: {
+          color: 'blue',
+          width: 2,
+        },
+        customdata: [],
+        hoverinfo: 'none',
+      },
```

Comment:
Single point trace with no custom data (otherwise plot.ly will not show the rectangle shape!)